fixbackend/subscription/models.py

```diff
@@ -52,4 +70,6 @@ class BillingEntry:
 
 
 # Multiple payment methods are possible, but for now we only support AWS Marketplace
-SubscriptionMethod = Union[AwsMarketplaceSubscription]
+SubscriptionMethod = Union[AwsMarketplaceSubscription, StripeSubscription]
+
+SubscriptionMethodType = TypeVar("SubscriptionMethodType", bound=SubscriptionMethod)
```

Comment:
Do we need this generic here? isn't union above enough?

Reply:
No: when I give you a StripeSubscription, I also want to receive one - not a generic payment method.